runtime/helpers/waitHelpers.js

```diff
@@ -316,8 +336,8 @@ module.exports = {
 	waitUntilEmailIsSent,
 	waitUntilUrlContains,
 	waitUntilUrlNotContains,
-	waitUntilAjaxIsFinished,
 	waitUntilPageLoads,
+	waitUntilLegacyPageLoads,
```

Comment:
I'm wondering, if we add waitUntilLegacyPageLoads to module.exports of waitHelpers.js, why don't we add waitUntilNuxtClientLoads as well?

Reply:
Nevermind, found it in line 346.